File: golang/04_add_2_numbers/add2.go
package main

import "fmt"

func main() {
	var var1, var2 int;
	fmt.Print("Enter a number: ")
	_, _ = fmt.Scanf("%d", &var1)
	fmt.Print("Enter another number: ")
	_, _ = fmt.Scanf("%d", &var2)
	fmt.Println("The sum is ", var1 + var2)
}

File: rust/04_add_2_numbers/add2.rs
fn main()
{
    println!("Enter a number: ");
    let mut line1  = String::new();
    std::io::stdin().read_line(&mut line1).unwrap();
    let number1 : i32 = line1.trim().parse().unwrap();

    println!("Enter another number: ");
    let mut line2  = String::new();
    std::io::stdin().read_line(&mut line2).unwrap();
    let number2 : i32 = line2.trim().parse().unwrap();

    println!("The sum is: {}", number1 + number2);
    
}